NSS-Spontaneous-Spoonbills/Sprint2
# Em morecompviews

## Description of Proposed Changes

* Added computer detail view and template

Outline the steps to test

```sh
git fetch --all
git checkout em_morecompviews
```

1. Start virtual environment and server
1. Run the following in the command line (if you haven't already):

```
python manage.py makemigrations WorkforceManagement
python manage.py migrate
```

1. In DB browser, open the database and manually enter at least two computers
1. Navigate to http://127.0.0.1:8000/WorkforceManagement/computers/
1.  Verify that the details of each computer are visible when you click on their links

## Impacted Areas in Application

**Modified Files**

README.md
WorkforceManagement/templates/WorkforceManagement/Computer_Detail.html
WorkforceManagement/templates/WorkforceManagement/Computer_List.html
WorkforceManagement/urls.py
WorkforceManagement/views/Computer_View.py

## Mentions @username

@binauralbeat @obscure-shadow @JacobAndrewSmith92 @kimberly-bird 

## Testing
